src/pudl/etl/__init__.py

```diff
@@ -34,6 +34,7 @@
 default_assets = (
     *load_assets_from_modules([eia_bulk_elec_assets], group_name="core_eia_bulk_elec"),
     *load_assets_from_modules([epacems_assets], group_name="core_epacems"),
```

Comment:
Captured the recent addition of the `core_` prefix here in the resolution of the last merge conflict.